modules/client-configuration/pages/clients-rh-cdn.adoc

```diff
@@ -4,7 +4,7 @@
 If you are running {rhel} clients directly, rather than using {sleses}, you need to use Red Hat sources to retrieve and update packages.
 This section contains information about using the {redhat} content delivery network (CDN) to register traditional and Salt clients running {rhel} operating systems.
 
-Traditional clients are available on {rhel}{nbsp}6 and 7 only.
+Traditional clients are available on {rhel}{nbsp} 7 only.
```

Comment:
```suggestion
Traditional clients are available on {rhel}{nbsp}7 only.
```
A space after the non-breaking space macro contradicts its usage.  I hope it is clear what I mean ;-)

Maybe, we need a vale check that reminds us, @jcayouette ?